evm_arithmetization/src/cpu/kernel/interpreter.rs

```diff
@@ -1215,264 +732,6 @@ impl<'a, F: Field> Interpreter<'a, F> {
-        }
-
-        if !self.is_kernel() {
-            self.add_jumpdest_offset(offset);
-        }
-
-        Ok(())
-    }
-
-    fn run_push(&mut self, num_bytes: u8) -> anyhow::Result<(), ProgramError> {
-        let x = U256::from_big_endian(&self.code_slice(num_bytes as usize));
-        self.incr(num_bytes as usize);
-        self.push(x)
-    }
-
-    fn run_dup(&mut self, n: u8) -> anyhow::Result<(), ProgramError> {
-        let len = self.stack_len();
-        if !self.is_kernel() && len >= MAX_USER_STACK_SIZE {
-            return Err(ProgramError::StackOverflow);
-        }
-        if n as usize > self.stack_len() {
-            return Err(ProgramError::StackUnderflow);
-        }
-        self.push(stack_peek(&self.generation_state, n as usize - 1)?)
-    }
-
-    fn run_swap(&mut self, n: u8) -> anyhow::Result<(), ProgramError> {
-        let len = self.stack_len();
-        if n as usize >= len {
-            return Err(ProgramError::StackUnderflow);
-        }
-        let to_swap = stack_peek(&self.generation_state, n as usize)?;
-        let old_value = self.stack_segment_mut()[len - n as usize - 1];
-
-        self.stack_segment_mut()[len - n as usize - 1] = self.stack_top()?;
-        let mem_write_op = InterpreterMemOpKind::Write(
-            old_value,
-            self.context(),
-            Segment::Stack.unscale(),
-            len - n as usize - 1,
-        );
-        self.memops.push(mem_write_op);
-        self.generation_state.registers.stack_top = to_swap;
-        Ok(())
-    }
-
-    fn run_get_context(&mut self) -> anyhow::Result<(), ProgramError> {
-        self.push(U256::from(self.context()) << CONTEXT_SCALING_FACTOR)
-    }
-
-    fn run_set_context(&mut self) -> anyhow::Result<(), ProgramError> {
-        let x = self.pop()?;
-        let new_ctx = u256_to_usize(x >> CONTEXT_SCALING_FACTOR)?;
-        let sp_to_save = self.stack_len().into();
-
-        let old_ctx = self.context();
-
-        let sp_field = ContextMetadata::StackSize.unscale();
-
-        let old_sp_addr = MemoryAddress::new(old_ctx, Segment::ContextMetadata, sp_field);
-        let new_sp_addr = MemoryAddress::new(new_ctx, Segment::ContextMetadata, sp_field);
-        self.generation_state.memory.set(old_sp_addr, sp_to_save);
-
-        let new_sp = u256_to_usize(self.generation_state.memory.get(new_sp_addr))?;
-
-        if new_sp > 0 {
-            let new_stack_top = self.generation_state.memory.contexts[new_ctx].segments
-                [Segment::Stack.unscale()]
-            .content[new_sp - 1];
-            self.generation_state.registers.stack_top = new_stack_top;
-        }
-        self.set_context(new_ctx);
-        self.generation_state.registers.stack_len = new_sp;
-
-        Ok(())
-    }
-
-    fn run_mload_general(&mut self) -> anyhow::Result<(), ProgramError> {
-        let addr = self.pop()?;
-        let (context, segment, offset) = unpack_address!(addr);
```

Comment:
I think the `unpack_address` macro can be deleted as all methods needing it have been removed.